### PR TITLE
fix: Unique to Snyk Highest Severity Vulnerability text issue

### PR DESCRIPTION
**Pull request by mathur07**
Issue :
image

Fix :
image

### CODE REVIEW

**Comment by mathur07** [comment]
[test]

**Comment by mathur07** [comment]
@arunkumars08 when using old plugin against new SA-UI this issue occurred.

File: src/app/stack/card-details/component-information/component-information.component.html [hunk 1]
```diff
@@ -62,9 +62,10 @@
                             <span *ngIf="comp?.hasSecurityIssue && tabType == 'private'">
                                 <a  [href]="comp?.privateSecurityDetails?.highestIssue?.url" target="_blank" class="name" title="command/ctrl + click"
                                     [innerText]="comp?.privateSecurityDetails && comp?.privateSecurityDetails?.highestIssue?.cve || '-'"></a>
-                                    <span *ngIf="comp?.registrationStatus !== 'registered'">
-                                        to <br> learn about the vulnerabilities found
-                                    </span>
+                                    
+                                <span *ngIf="comp?.registrationStatus !== 'registered' && comp?.privateSecurityDetails && comp?.privateSecurityDetails?.highestIssue?.cve">
```
Comment (arajkumar) [inline src/app/stack/card-details/component-information/component-information.component.html:66]:
```suggestion
                                <span *ngIf="comp?.registrationStatus !== 'registered' &&  comp?.privateSecurityDetails?.highestIssue?.cve">
```

**Review by arajkumar — CHANGES_REQUESTED** [review]

**Review by arajkumar — APPROVED** [review]
lgtm

**Comment by mathur07** [comment]
[test]

**Comment by mathur07** [comment]
[test]

**Comment by mathur07** [comment]
[test]

**Comment by mathur07** [comment]
[test]

**Comment by mathur07** [comment]
[test]

**Comment by mathur07** [comment]
[test]

**Comment by mathur07** [comment]
[test]

**Comment by mathur07** [comment]
[test]

**Comment by mathur07** [comment]
[test]

**Comment by mathur07** [comment]
[test]

**Comment by mathur07** [comment]
[test]

**Comment by mathur07** [comment]
[test]

**Pull request merged**